core/src/main/java/bio/terra/pearl/core/service/survey/SurveyParseUtils.java

```diff
@@ -110,7 +152,17 @@ public static String unmarshalSurveyQuestionChoices(JsonNode question) {
             if (choice.isTextual()) {
                 choices.add(new QuestionChoice(choice.asText(), choice.asText()));
             } else {
-                choices.add(new QuestionChoice(choice.get("value").asText(), choice.get("text").asText()));
+                JsonNode textNode = choice.get("text");
+
+                String text;
+                if (textNode.isTextual()) {
+                    text = textNode.asText();
+                } else {
+                    // todo: make this grab default language
+                    text = textNode.get("en").asText();
+                }
+
+                choices.add(new QuestionChoice(choice.get("value").asText(), text));
```

Comment:
Drive by fix: any questions that have text that was translated into multiple languages would have QuestionChoices with empty text strings, since we were trying to turn a json object into a string and it just doesn't do it